Source/WebKit/UIProcess/GeolocationPermissionRequestManagerProxy.cpp

```diff
@@ -49,7 +49,7 @@ void GeolocationPermissionRequestManagerProxy::invalidateRequests()
 
 Ref<GeolocationPermissionRequestProxy> GeolocationPermissionRequestManagerProxy::createRequest(GeolocationIdentifier geolocationID, WebProcessProxy& process)
 {
-    auto request = GeolocationPermissionRequestProxy::create(this, geolocationID, process);
+    auto request = GeolocationPermissionRequestProxy::create(*this, geolocationID, process);
```

Comment:
Just a nit: I believe we prefer to use `Ref` instead of `auto` where possible. Maybe we can do that here?

Reply:
Yes indeed. Will fix.